src/prefect/engine.py

```diff
@@ -1607,8 +1617,11 @@ async def report_task_run_crashes(task_run: TaskRun, client: OrionClient):
     except (Abort, Pause):
         # Do not capture internal signals as crashes
         raise
+    except Cancel:
+        # Do not capture cancellations as crashes
+        raise
     except BaseException as exc:
-        state = await exception_to_crashed_state(exc)
+        state = await exception_to_final_state(exc)
```

Comment:
Here you ignore `Cancel` but you added handling of `Cancel` to `exception_to_final_state` in https://github.com/PrefectHQ/prefect/pull/7887/files#diff-d96d5698f54cffee300418df379152a03d0f51afedb400bb59849549d95372eeR138-R140 ?

Reply:
Right, this should probably not capture those.